### PR TITLE
DM-29110: Add a pipeline for fakes ci

### PR DESCRIPTION
**Pull request by sr525**

### CODE REVIEW

File: bin/pipeline.sh [hunk 1]
```diff
@@ -25,3 +28,16 @@ pipetask run -j "$1" -b "$2"/butler.yaml \
     --input "$INPUTCOLL" --output "$COLLECTION" \
     --register-dataset-types \
     --qgraph "$QGRAPH_FILE"
+
```
Comment (MorganSchmitz) [inline bin/pipeline.sh:31]: Can I just suggest copy/pasting the #TODO comment above this line? Just so we can make sure the exclusion is removed in the fakes run as well whenever DM-29751 gets solved.

File: resources/external_fakes.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - skymap
+  - tract
+  storage_class: DataFrame
+  is_calibration: false
+- type: dataset
+  dataset_type: fakes_fakeSourceCat
+  run: HSC/external
+  records:
+  - dataset_id:
+    - 9999
```
Comment (MorganSchmitz) [inline resources/external_fakes.yaml:16]: I confess I have no idea what these external yaml files do. I'm guessing 9999 here is just so there's no conflict with other `dataset_id`s?

**Review by MorganSchmitz — APPROVED** [review]

File: resources/external_fakes.yaml (hunk 2 above)
Comment (timj) [reply resources/external_fakes.yaml:16]: ci_hsc_gen3 now uses UUID as the dataset IDs so 9999 looks very different to the normal expectation.

Comment (sr525) [reply resources/external_fakes.yaml:16]: I picked a random number. The external yaml files allow you to specify files that are needed but not created in previous tasks.

Comment (sr525) [reply resources/external_fakes.yaml:16]: I can use anything else instead if that will be better?

Comment (timj) [reply resources/external_fakes.yaml:16]: If this is going to be imported with butler import I'm pretty certain that it doesn't matter what the number is in the file so long as within the file it's self-consistent. We'll rewrite on import. You can test that by doing a `butler query-datasets` and looking at the ID after import.

**Pull request merged**